addons/html_editor/static/src/core/selection_plugin.js

```diff
@@ -210,6 +210,9 @@ export class SelectionPlugin extends Plugin {
             }
         });
         this.addDomListener(this.editable, "mousedown", (ev) => {
+            if (!ev.target.isContentEditable) {
```

Comment:
We might need another check to see if the mouse down target is in editable? to avoid that the popover is in editing mode, then it won't close if we click on the contenteditable false element (the mega menu thing)